rules/integrations/endpoint/elastic_endpoint_security_behavioral.toml

```diff
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2021/09/18"
+maturity = "production"
+updated_date = "2021/09/18"
+integration = "endpoint"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Endpoint Security alert is received for Behavior Protection alerts.
+Enabling this rule allows you to immediately begin investigating your Endpoint alerts for Behavior Protection.
+"""
+enabled = true
+from = "now-10m"
+index = ["logs-endpoint.alerts-*"]
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 10000
+name = "Endpoint Security Behavioral Protection"
+risk_score = 47
```

Comment:
might be able to use `risk_score_mapping` and `severity_mapping` here to inherit from the endpoint rule

Reply:
disregard, now I see below 😄